VirtoCommerce.Storefront.Model/VirtoCommerce.Storefront.Model.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
not sure that right use .Web, it is a library.

Reply:
You're right. I made changes.